### PR TITLE
AbuseDB - Packaging

### PR DESCRIPTION
**Pull request by orlichter1**
## Status
Ready

## Description
Just packaging of AbuseDB

### CODE REVIEW

**Comment by Itay4** [comment]
image is too big (more than 4kb), let's get one according to convention

**Comment by orlichter1** [comment]
Eynav said that we can use this file even if it's 4.7 kb

File: Integrations/AbuseDB/AbuseDB.yml [hunk 1]
```diff
@@ -0,0 +1,187 @@
+commonfields:
+  id: AbuseIPDB
+  version: -1
+name: AbuseIPDB
+display: AbuseIPDB
+category: Data Enrichment & Threat Intelligence
+description: Central repository to report and identify IP addresses that have been
+  associated with malicious activity online. Check the Detailed Information section
+  for more information on how to configure the integration.-
```
Comment (yaakovi) [inline Integrations/AbuseDB/AbuseDB.yml]:
```suggestion
  for more information on how to configure the integration.
```

**Comment by yaakovi** [comment]
in the following PR, let alex go over the entire yml file, it is not up to standard

**Comment by orlichter1** [comment]
@yaakovi It failed because of the subtype. could you please approve :)

**Pull request merged**